src/gobu/main.go

```diff
@@ -56,6 +56,11 @@ func runShell(version string) {
 
 	log.Println(">> You are now in a new GOBU shell. To exit, type 'exit'")
 	defaultShell := resolveBinary(os.Getenv("SHELL"))
+	
+	if runtime.GOOS == "windows" {
+	
+		defaultShell = "cmd.exe" // TODO find parent process name
```

Comment:
thanks for the advice, i am getting the used parent process per go-ps now and execute it in a windows environment